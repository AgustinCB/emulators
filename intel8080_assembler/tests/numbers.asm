MVI A, 10
MVI A, 10N
MVI A, 10O
MVI A, 10H
JMP $ + 1
JMP 4+3