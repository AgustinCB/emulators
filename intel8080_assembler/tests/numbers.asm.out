0000 MVI A,#$0a
0002 MVI A,#$02
0004 MVI A,#$08
0006 MVI A,#$10
0008 JMP $0009
000b JMP $0007
000e JMP $0061
0011 JMP $0001
0014 JMP $0002
0017 JMP $0002
001a JMP $0004
001d JMP $0019
0020 JMP $0001
0023 JMP $0002
0026 JMP $000a
0029 JMP $0002
002c JMP $0001
002f JMP $0007
0032 JMP $0006
0035 JMP $ffff
0038 JMP $0006
003b JMP $0008
003e JMP $ffff
